CruCentralCoast/Database/CommunityGroup.swift

```diff
@@ -32,6 +32,7 @@ class CommunityGroup: RealmObject {
     var gender: Gender { return Gender(rawValue: self.genderString) ?? .mixed }
     var year: Year { return Year(rawValue: self.yearString) ?? .mixed }
     var weekDay: WeekDay { return WeekDay(rawValue: self.weekDayString) ?? .sunday }
+    var leaderNames: String? { return self.leaders.compactMap { $0.name }.joined(separator: ", ") }
```

Comment:
Also out of curiosity, what does the "$0" syntax mean?

Reply:
Some languages call this an "anonymous function", others call it a "closure" (Swift uses "closure"). It acts like a function in that it takes in parameters and returns one or more values. The style I am using is a shortened form of a closure, where each parameter is not named, but can be accessed via $0, $1, $2, etc. The return value is whatever I put in the closure { }.

Reply:
cool thanks!